plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationContext.java

```diff
@@ -30,6 +30,8 @@
 
     private transient final ConfiguratorRegistry registry;
 
+    private String mode;
```

Comment:
what's this mode value for? needs docs and probably should be an enum if required

Reply:
Its just a mode to tell if the context is jsonSchema

Reply:
should be an enum or a boolean really

Reply:
Yeah will incorporate it in the next set of changes.